src/model/deserialized.rs

```diff
@@ -15,6 +16,7 @@ pub enum BinXMLDeserializedTokens<'a> {
     CloseEmptyElement,
     CloseElement,
     Value(BinXmlValue<'a>),
+    RefValue(&'a BinXmlValue<'a>),
```

Comment:
Perhaps we are missing documentation for this enum, but this new variant deviates from what glues the enum members together.

It seems like Value and RefValue are some sort of `Cow<‘a, Value<‘a>>`, could you explain better why this is needed?